Makefile

```diff
@@ -1,33 +1,36 @@
+DART_DEV=https://dev.dart.harvard.edu
```

Comment:
What's the advantage of having multiple vars here? Could we just use a single var, and pull its value from an environment variable like `DART_UI_TEST_URL_BASE` or something like that, and have e.g. `https://dev.dart.harvard.edu` as a default if there is no environment variable override defined?